app/src/ui/changes/changes-list.tsx

```diff
@@ -377,6 +379,30 @@ export class ChangesList extends React.Component<
     showContextualMenu(items)
   }
 
+  private getNamesOfSelectedOversizedFiles = () => {
+    const fileNames: string[] = []
+    const workingDirectoryFiles = this.props.workingDirectory.files
+
+    for (const file of workingDirectoryFiles) {
+      const fileIsSelected =
+        file.selection.getSelectionType() !== DiffSelectionType.None
+
+      if (fileIsSelected) {
+        try {
+          const filePath = path.join(this.props.repository.path, file.path)
+          // eslint-disable-next-line no-sync
+          const fileStatus = FSE.statSync(filePath)
```

Comment:
`fs-extra` has a promise-based API, so this can be rewritten to avoid making `eslint` annoyed:

```ts
const fileStatus = await FSE.stat(filePath)
```